metadata-ingestion/src/datahub/ingestion/source/powerbi/config.py

```diff
@@ -139,6 +140,10 @@ class PowerBiAPIConfig(EnvBasedSourceConfigBase):
     extract_lineage: bool = pydantic.Field(
         default=True, description="Whether lineage should be ingested"
     )
+    # Enable/Disable extracting endorsements to tags
+    extract_endorsements_to_tags: bool = pydantic.Field(
+        default=True, description="Whether to extract endorsements to tags"
```

Comment:
Can we default this to "False"? Most people may be surprised by this behavior, since this can overwrite existing Tags defined in DataHub